src/components/CustomerQuota/CheckoutSuccess/CheckoutSuccessCard.tsx

```diff
@@ -40,7 +40,7 @@ const styles = StyleSheet.create({
 interface CheckoutSuccessCard {
   ids: string[];
   onCancel: () => void;
-  quotaResponse: Quota | null;
+  quotaResponse?: Quota;
```

Comment:
Hmmm minor distinction, by having it as `?` we imply that the quotaResponse can be excluded, which isn't the case. Having it as `quotaResponse: Quota | undefined` implies instead that we *must* pass in a `quotaResponse` just that it could possibly be undefined. This prevents future errors when `CheckoutSuccessCard` is used but no `quotaResponse` was passed in.

Reply:
I agree with this. This is done!